.github/workflows/buildAll.yml

```diff
@@ -1,6 +1,10 @@
 name: Build All
 on:
   workflow_call:
+    inputs:
+      branch:
+        type: string
+        required: false
```

Comment:
note that required is set to false so that we did not have to update the commit workflow and it will still default to the associated branch